scripts/python/SLiM_sims.py

```diff
@@ -47,25 +87,72 @@ def bgzip(outpath):
     print(err)
 
 
+def tabix_vcfs(outpath):
+
+    # Use find utility to identify all VCFs in outpath
+    process_find = find_vcfs(outpath, 'vcf.gz')
+
+    # bgzip all found VCFs
+    process_tabix = subprocess.Popen(['xargs', '-n1', 'tabix', '-f'],
+                                     stdin=process_find.stdout,
+                                     stderr=subprocess.PIPE,
+                                     universal_newlines=True)
+
+    out, err = process_tabix.communicate()
+
+    # print(out)
+    print(err)
+
+
+def vcf2fasta(table, region, mut_mat, outpath):
+
+    fasta_outpath = outpath + "fasta-files/"
+    create_output_directory(fasta_outpath)
+
+    for vcf in tqdm(glob(outpath + '*.vcf.gz')):
+
+        filename = fasta_outpath + vcf.split('/')[-1].split('.vcf.gz')[0] + ".fasta"
+
+        slim_vcf2fasta_chrom.write_fasta(vcf, table, region, mut_mat, filename)
```

Comment:
Leaving note here from in-person meeting - look for ways to incorporate `multiprocessing` with chunks of filenames to speed up vcf -> fasta processing